docs/InstallGuide.md

```diff
@@ -1,424 +1,358 @@
-# OpenVNet #
 
-# Preliminary Operations and Installation #
+# OpenVNet Installation Guide
 
-> **Note:**
-This document explains the installation of OpenVNet.  However, it was
-written for an older version of OpenVNet and does not apply
-to the current version.  We plan to update this guide soon.  Until
-then, we are leaving this guide here and hope it will be useful for those
-who wish to learn more about the components that make up OpenVNet.
+## Overview
 
 
-Installation Requirements
--------------------------
+Welcome to the world of OpenVNet! With this installation guide we help you
+create a very simple yet innovative virtual network environment.
 
-### System Requirements
+![vnet_minimum](https://www.dropbox.com/s/dezarv561fg7sdj/vnet_minimum.png?dl=1)
+
+On a given server (here named as server1) there is one virtual network whose network address is 10.100.0.0/24 and 2 virtual machines joining it.
+The orange circles describe OpenVNet's ruby processes; vna, vnmgr and webapi.
+See architecture for more detail of how the OpenVNet works.
+
+
+## Requirements
 
-+ RHEL 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
 
 ### Network Requirements
 
 + Local Area Network (LAN)
 + Internet connection
 
-Pre-setup
----------
+## Installation
 
-Download the openvnet.repo file and put it to your /etc/yum.repos.d/ directory.
+### Install OpenVNet Packages
+
+Download the openvnet.repo file and put it to your `/etc/yum.repos.d/` directory.
 
     # curl -o /etc/yum.repos.d/openvnet.repo -R https://raw.githubusercontent.com/axsh/openvnet/master/openvnet.repo
 
-Download the openvnet-third-party.repo file and put it in your /etc/yum.repos.d/ directory.
+Download the openvnet-third-party.repo file and put it in your `/etc/yum.repos.d/` directory.
 
     # curl -o /etc/yum.repos.d/openvnet-third-party.repo -R https://raw.githubusercontent.com/axsh/openvnet/master/openvnet-third-party.repo
 
-Install epel-release.
+Each repo has the following packages:
 
-    # rpm -Uvh http://dlc.wakame.axsh.jp.s3-website-us-east-1.amazonaws.com/epel-release
+* openvnet.repo
+  * openvnet (metapackage)
+  * openvnet-common
+  * openvnet-vna
+  * openvnet-vnmgr
+  * openvnet-webapi
```

Comment:
I see lower in the guide that vnctl is expected to work out of the box. That means the guide is geared towards the 0.7 release which is the correct thing to do. That also means `openvnet-vnctl` should be added to this list.